grafana/scylla-cql.template.json

```diff
@@ -231,6 +231,24 @@
             },
             {
                 "class": "row",
+                "height": "25px",
+                "gridPos": {"h": 3},
+                "dashproductreject": "no-cql-connection",
+                "panels": [
+                    {
+                        "class": "text_panel",
+                        "options": {
+                            "content": "<span style=\"color:#5881c2;font-size: 2rem;\">CQL System tables</span></br><span style=\"\">The following information is based on Scylla Plugin configuration, check the documentation for more details on how to enable it.</span>",
```

Comment:
which documentation?
why not adding a direct link, or at least URL

Reply:
please review the documentation pr and I'll add it